lib/beaker/host/windows/pkg.rb

```diff
@@ -101,15 +89,7 @@ def pe_puppet_agent_promoted_package_info( puppet_collection = nil, opts = {} )
 
   # @api private
   def identify_windows_architecture
-    arch = nil
-    execute("echo '' | wmic os get osarchitecture", :accept_all_exit_codes => true) do |result|
-      arch = if result.exit_code == 0
-        result.stdout =~ /64/ ? '64' : '32'
-      else
-        identify_windows_architecture_from_os_name_for_win2003
-      end
-    end
-    arch
+    platform.arch =~ /64/ ? '64' : '32'
```

Comment:
3rd parties can still name images whatever they like; however, platform strings supplied to beaker should still be split on the `variant`-`version`-`arch`.

In this case, if beaker relies on the robust method, you may end up testing the wrong architecture underneath and you would never know unless you were carefully inspecting your testing logs.